packages/genesys-spark-components/src/components/stable/gux-table/example.html

```diff
@@ -438,11 +438,26 @@ <h2>Data table with sortable columns</h2>
 </gux-table>
 
 <h2>Object table with rows selection</h2>
+<ul>
+  <li>
+    Use the <strong>setChecked</strong> method on
+    <strong>gux-all-row-select</strong> to programatically manage the selected
+    state.
+  </li>
+  <li>
+    Use the <strong>setIndeterminate</strong> method to set the
+    indeterminate/unknown state on <strong>gux-all-row-select</strong>. Remember
+    to set it to false when the checkbox is checked or unchecked as it overrides
+    these states.
+  </li>
+</ul>
+<gux-button id="selectAll">Select all</gux-button>
+<gux-button id="deselecAll">Deselect all</gux-button>
 <gux-table object-table selectable-rows>
   <table slot="data">
     <thead>
       <tr data-row-id="head">
-        <th><gux-all-row-select></gux-all-row-select></th>
+        <th><gux-all-row-select id="allRowSelect"></gux-all-row-select></th>
```

Comment:
I think it might be easier to just query select the `gux-all-row-select` element instead of adding an `id`.

Reply:
That would select all the `gux-all-row-select`'s on the page, I just want to select that particular one as it relates to one particular table.